code/modules/research/message_server.dm

```diff
@@ -115,65 +115,6 @@ var/global/list/obj/machinery/message_server/message_servers = list()
 
 	return
 
-
-/datum/feedback_variable
-	var/variable
-	var/value
-	var/details
-
-/datum/feedback_variable/New(var/param_variable,var/param_value = 0)
-	variable = param_variable
-	value = param_value
-
-/datum/feedback_variable/proc/inc(var/num = 1)
-	if(isnum(value))
-		value += num
-	else
-		value = text2num(value)
-		if(isnum(value))
-			value += num
-		else
-			value = num
-
-/datum/feedback_variable/proc/dec(var/num = 1)
-	if(isnum(value))
-		value -= num
-	else
-		value = text2num(value)
-		if(isnum(value))
-			value -= num
-		else
-			value = -num
-
-/datum/feedback_variable/proc/set_value(var/num)
-	if(isnum(num))
-		value = num
-
-/datum/feedback_variable/proc/get_value()
-	return value
-
-/datum/feedback_variable/proc/get_variable()
-	return variable
-
-/datum/feedback_variable/proc/set_details(var/text)
-	if(istext(text))
-		details = text
-
-/datum/feedback_variable/proc/add_details(var/text)
-	if(istext(text))
-		if(!details)
-			details = text
-		else
-			details += " [text]"
-
-/datum/feedback_variable/proc/get_details()
-	return details
-
-/datum/feedback_variable/proc/get_parsed()
-	return list(variable,value,details)
-
-var/obj/machinery/blackbox_recorder/blackbox
-
 /obj/machinery/blackbox_recorder
```

Comment:
So does this thing even do anything anymore?
Also holy fuck

Reply:
nope, it just exists

Reply:
Can't you bind some of the blackbox functionality to this object? So that it doesn't work if it's destroyed or something?

Reply:
I'm just going to pretend you didn't say that

Reply:
@DeityLink For what possible reason could that be even remotely desirable?